cli/config/config.go

```diff
@@ -0,0 +1,13 @@
+package config
```

Comment:
I think so, but this will require more refactoring. I'd like to do that in a separate PR if that is acceptable?